cmd/log.go

```diff
@@ -42,6 +42,7 @@ var logCmd = &cobra.Command{
 				"namespaces",
 				"services",
 				"nodes",
+				"pods",
```

Comment:
You should also implement this case in the `ListerFor` function to have a working completion (called 5 lines below).

Reply:
Done